api/Models/AuthOptions.cs

```diff
@@ -1,19 +1,14 @@
-﻿using Microsoft.AspNetCore.Mvc;
-using Microsoft.IdentityModel.Tokens;
-using System;
-using System.Collections.Generic;
-using System.Linq;
+﻿using Microsoft.IdentityModel.Tokens;
 using System.Text;
-using System.Threading.Tasks;
 
-namespace api.Models
+namespace iTechArt.Hotels.Api.Models
 {
     public class AuthOptions
     {
         public string Issuer { get; set; }
         public string Audience { get; set; }
         public string Secret { get; set; }
-        public int TokenLifeTime { get; set; } //secs
+        public int ExpireTime { get; set; }
```

Comment:
How is it better from previous version? I still don't understand what value should I put here.